examples/AlwaysOnTraceExample.php

```diff
@@ -3,13 +3,13 @@
 declare(strict_types=1);
 require __DIR__ . '/../vendor/autoload.php';
 
-use OpenTelemetry\Sdk\Trace\AlwaysOnSampler;
+use OpenTelemetry\Contrib\Zipkin\Exporter;
```

Comment:
When I ran the examples locally with this branch, I got an error (included below) - I think we need to maybe `use` this as another name, because it's conflicting with `sdk/Trace/Exporter.php` - Maybe we should import it `as ZipkinExporter` or something like that?

```
bentest ? make examples                                                                                                                                                                                                                                                                                                                                                   2 ↵
docker-compose up -d
Starting opentelemetry-php_php_1 ...
opentelemetry-php_zipkin_1 is up-to-date
Starting opentelemetry-php_php_1 ... done
docker-compose run --rm php php ./examples/AlwaysOnTraceExample.php

Fatal error: Uncaught Error: Class 'OpenTelemetry\Contrib\Zipkin\Exporter' not found in /usr/src/myapp/examples/AlwaysOnTraceExample.php on line 24

Error: Class 'OpenTelemetry\Contrib\Zipkin\Exporter' not found in /usr/src/myapp/examples/AlwaysOnTraceExample.php on line 24

Call Stack:
    0.0011     405464   1. {main}() /usr/src/myapp/examples/AlwaysOnTraceExample.php:0

make: *** [examples] Error 255```

Reply:
I imported Zipking/Exporter to ZipkinExporter. I couldn't reproduce this error before, but can you try it now and see if it works?

For me it works ok. I also tried to remove all library directory and reinstall it, but I couldn't reproduce the error.

```bash
Beniamins-MacBook➜  opentelemetry : organized-trace-sdk ✔ : ᐅ  make examples                                                                                                                                                      (emag-prod-admin@content-prod/content)
docker-compose up -d
Creating opentelemetry_php_1    ... done
Creating opentelemetry_zipkin_1 ... done
Creating opentelemetry_jaeger_1 ... done
docker-compose run --rm php php ./examples/AlwaysOnTraceExample.php
Starting AlwaysOnTraceExample
Trace with id 00609987d713d0f9e434a7fbf10513b5 started 
AlwaysOnTraceExample complete!  See the results at http://localhost:9411/
docker-compose run --rm php php ./examples/AlwaysOffTraceExample.php

Sampling is not enabled
docker-compose run --rm php php ./examples/JaegerExporterExample.php
Starting JaegerExporterExample
Trace with id 3e8da8026b581b89f508589ef8c97f73 started [{"id":"338a016600bcb55e","traceId":"3e8da8026b581b89f508589ef8c97f73","parentId":"6204daafba5ee7fe","localEndpoint":{"serviceName":"jaegerExporterExample"},"name":"session.generate.span1590676668","timestamp":1590676668901991,"duration":114021120,"tags":{"remote_ip":"1.2.3.4","country":"USA"},"annotations":[{"timestamp":1590676668901991,"value":"found_login0"},{"timestamp":1590676668901991,"value":"generated_session"}]}][{"id":"33b0c7074625d1db","traceId":"3e8da8026b581b89f508589ef8c97f73","parentId":"6204daafba5ee7fe","localEndpoint":{"serviceName":"jaegerExporterExample"},"name":"session.generate.span1590676669","timestamp":1590676669934031,"duration":928000,"tags":{"remote_ip":"1.2.3.4","country":"USA"},"annotations":[{"timestamp":1590676669934031,"value":"found_login1"},{"timestamp":1590676669934031,"value":"generated_session"}]}][{"id":"690cde2c2ac341b6","traceId":"3e8da8026b581b89f508589ef8c97f73","parentId":"6204daafba5ee7fe","localEndpoint":{"serviceName":"jaegerExporterExample"},"name":"session.generate.span1590676669","timestamp":1590676669969406,"duration":760064,"tags":{"remote_ip":"1.2.3.4","country":"USA"},"annotations":[{"timestamp":1590676669969406,"value":"found_login2"},{"timestamp":1590676669969406,"value":"generated_session"}]}][{"id":"39e0f62e1516c569","traceId":"3e8da8026b581b89f508589ef8c97f73","parentId":"6204daafba5ee7fe","localEndpoint":{"serviceName":"jaegerExporterExample"},"name":"session.generate.span1590676670","timestamp":1590676670026732,"duration":581120,"tags":{"remote_ip":"1.2.3.4","country":"USA"},"annotations":[{"timestamp":1590676670026732,"value":"found_login3"},{"timestamp":1590676670026732,"value":"generated_session"}]}][{"id":"57441e6a26d91f61","traceId":"3e8da8026b581b89f508589ef8c97f73","parentId":"6204daafba5ee7fe","localEndpoint":{"serviceName":"jaegerExporterExample"},"name":"session.generate.span1590676670","timestamp":1590676670069244,"duration":4576000,"tags":{"remote_ip":"1.2.3.4","country":"USA"},"annotations":[{"timestamp":1590676670069244,"value":"found_login4"},{"timestamp":1590676670069244,"value":"generated_session"}]}]
JaegerExporterExample complete!  See the results at http://localhost:16686/
Beniamins-MacBook➜  opentelemetry : organized-trace-sdk ✔ : ᐅ                                              
```

Reply:
Yup; this worked for me now too.